blockstack-sdk/src/main/java/org/blockstack/android/sdk/ui/ConnectHowItWorksActivity.kt

```diff
@@ -0,0 +1,46 @@
+package org.blockstack.android.sdk.ui
+
+import android.content.Intent
+import android.graphics.BitmapFactory
+import android.net.Uri
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.browser.customtabs.CustomTabsIntent
+import androidx.core.content.ContextCompat
+import kotlinx.android.synthetic.main.activity_connect_how_it_works.*
+import org.blockstack.android.sdk.BlockstackSignIn
+import org.blockstack.android.sdk.R
+
+class ConnectHowItWorksActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_connect_how_it_works)
+
+        button_get_started.setOnClickListener {
+            setResult(RESULT_OK)
+            this.finish()
+        }
+
+        power_by_blockstack.setOnClickListener {
+            val locationUri = Uri.parse("https://www.blockstack.org/")
+            if (BlockstackSignIn.shouldLaunchInCustomTabs) {
+                val builder = CustomTabsIntent.Builder()
+                val options = BitmapFactory.Options()
+                options.outWidth = 24
+                options.outHeight = 24
+                options.inScaled = true
+                val backButton = BitmapFactory.decodeResource(resources, R.drawable.ic_arrow_back, options)
+                builder.setCloseButtonIcon(backButton)
```

Comment:
Isn't there a default back button? Do we really need all this?

Reply:
Default "back button" is a "close button", the cross one.